eng/packages/http-client-csharp/eng/pipelines/ci.yml

```diff
@@ -0,0 +1,67 @@
+trigger: none
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - eng/packages/http-client-csharp
+  
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: 'Build'
+        pool:
+          name: azsdk-pool-mms-win-2022-general
```

Comment:
Please include the variables from https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/variables/image.yml and reference the names of the pool and image via variables. Similar to https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/bicep-warnings.yml#L21

Reply:
updated